index.html

```diff
@@ -33,140 +40,210 @@ <h1>I’m Ismail Glad to see you!</h1>
         </section>
 
         <div id="container1">
-            <section class="work">
-                <img src="images/Snapshoot-Portfolio-1.png" alt="">
-                <h2>Tonic</h2>
-                <div class="counter">
-                    <span>CANOPY</span>
-                    <span class="counters"></span>
-                    <span>Back End Dev</span>
-                    <span class="counters"></span>
-                    <span>2015</span>
+            <section class="row work">
+                <div class="col-ss-7">
```

Comment:
- [ ] As mentioned earlier, kindly make sure you stick to the original design. the spacing, typos, ... are supposed to be similar to the original design.
- [ ] Also you may consider readjusting the size of your project images and cards as you may notice how big they are compared to the original design as well as their distance from the edge of the window.

## Your design :point_down: 
![Screenshot from 2023-05-12 15-18-27](https://github.com/yayaismaail/toolbar-headline/assets/97100091/613ada85-4dc4-48ca-827f-f8d038a8c747)

![Screenshot from 2023-05-12 15-22-36](https://github.com/yayaismaail/toolbar-headline/assets/97100091/40043e63-8150-45d6-9ec2-7f1900836c59)


## Orginal design :point_down: 
![Screenshot from 2023-05-12 15-07-08](https://github.com/yayaismaail/toolbar-headline/assets/97100091/66f9fa31-8fe8-4297-84c7-2cfeb74e60ed)

Reply:
- [ Optional ] As suggested by the previous reviewer Kindly adjust the alignment of the cards paragraph just like the Figma Design. 
- Your Design look likes this 
![Capture](https://github.com/yayaismaail/toolbar-headline/assets/93598189/35924899-fcc7-45df-b3cf-18e342c3b7bc)